test/spell-checker.js

```diff
@@ -78,6 +78,13 @@ eslintTester.addRuleTest('rules/spell-checker', {
             errors: [
                 { message: 'You have a misspelled word: tsih on String'},
                 { message: 'You have a misspelled word: srting on String'}]
+        },
+        {
+            code: 'var a = 1 // color is a comment normalize dict',
+            args:[2, {lang: 'en_GB', skipWords: ['dict']}],
+            errors: [
+                { message: 'You have a misspelled word: color on Comment'},
+                { message: 'You have a misspelled word: normalize on Comment'}]
```

Comment:
Yes! you have to install gulp
```
npm install -g gulp
```
and in the project folder for the first time:
```
npm install
```
then to run tests:
```
gulp test
```

You can even check for coverage in the ```coverage```folder